index.html

```diff
@@ -0,0 +1,154 @@
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      href="https://fonts.googleapis.com/css?family=Lato:400,700"
+      rel="stylesheet"
+      type="text/css"
+    />
+    <link
+      rel="stylesheet"
+      href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+    <link rel="stylesheet" href="assets/CSS/style.css" />
+    <link rel="stylesheet" href="assets/CSS/about.css" />
+    <title>Nutrition</title>
+  </head>
+  <body>
+    <div class="container-fluid">
+      <div class="row">
+        <nav class="navbar navbar-inverse text-white socialMediaIcons" id="home">
+          <div class="icon-bar navbar-right">
+            <a href="#" class="facebook"><i class="fa fa-facebook"></i></a>
+            <a href="#" class="twitter"><i class="fa fa-twitter"></i></a>
+            <a href="#">My page</a>
+            <a href="#">Logout</a>
+          </div>
+        </nav>
+      </div>
+      <div class="row">
+        <nav class="navbar" id="navbar">
+          <div class="navbar-header">
+            <a class="navbar-brand" href="#">
+              <img src="assets/images/img4.png" alt="navbar logo imag" />
+            </a>
+        </div>
+          <ul class="nav navbar-nav navItem text-secondary" id="navMenu">
+            <li class="active"><a href="index.html">Home</a></li>
+            <li><a href="about.html">About</a></li>
+            <li><a href="#program">Program</a></li>
+            <li><a href="#sports">Sports</a></li>
+            <li><a href="#">Sponser</a></li>
+            <li><a href="#">Nutrition</a></li>
+          </ul>
+         
+          <div class="humgurger">
+            <span class="bar"></span>
+            <span class="bar"></span>
+            <span class="bar"></span>
+          </div>
+        </nav>
+      </div>
+      <div class="row introduction">
+        
+        <p>"Welcome to Healthline Nutrition!"</p>
+        <h1 class="secondHeader">
+          The Definitive Guide<br />
+          to Healthy Eating in Real Life
+        </h1>
+
+        <p id="second-p">
+          healthy eating doesn’t have to be complicated. It’s entirely possible
+          to nourish your body while enjoying the foods you love.
+        </p>
+        <img src="assets/images/Health-Nutrition.jpg" alt="Nutrition" />
+        
+      </div>
+      <div class="row mainProgram" id="program">
+        <h3>Main Program</h3>
+        <hr />
+        <div class="col-sm card">
+          <img src="assets/images/vegtables.jpg" alt="Vegtables" />
+          <h4 class="card-title">Vegetables</h4>
+          <p class="card-text">
+            Add variety to grilled or steamed vegetables with an herb such as
+            rosemary..
```

Comment:
- [ ] Let's make this section even better! I would suggest the following to make your section as similar as possible to the template:
- Please add a 5th element to the list.
- The images should actually be icons.
- The line below the title should be orange.
- There's much empty space below the items description.
- The text alignment should be center.

|Template|Your design|
|---|---|
|<img width="921" alt="Screen Shot 2022-09-12 at 11 02 05 AM" src="https://user-images.githubusercontent.com/83717285/189701633-c34377ff-8850-4166-8a10-1cebb6953398.png">|<img width="1341" alt="Screen Shot 2022-09-12 at 10 34 34 AM" src="https://user-images.githubusercontent.com/83717285/189701712-0d091aee-9d04-434b-9cd1-91b3c6338b39.png">|

Reply:
- update the width , height  and margin 

- Add border on hover